alacritty_terminal/src/tty/windows/child.rs

```diff
@@ -85,6 +90,21 @@ impl ChildExitWatcher {
     pub fn deregister(&self) {
         *self.interest.lock().unwrap() = None;
     }
+
+    /// Retrieve the process handle of the underlying child process.
+    /// This function does **not** pass ownership of the raw handle to you,
+    /// and the handle is only guaranteed to be valid while the hosted application
+    /// has not yet been destroyed.
+    /// If you terminate the process using this handle, the terminal will get a
+    /// timeout error, and the child watcher will emit an `Exited` event.
```

Comment:
Just a bit less dense.

```suggestion
    /// Retrieve the process handle of the underlying child process.
    ///
    /// This function does **not** pass ownership of the raw handle to you, and the
    /// handle is only guaranteed to be valid while the hosted application has not
    /// yet been destroyed.
    ///
    /// If you terminate the process using this handle, the terminal will get
    /// timeout error, and the child watcher will emit an `Exited` event.
```

Reply:
Done.